teammapper-frontend/mmp/src/map/handlers/history.ts

```diff
@@ -120,8 +125,11 @@ export default class History {
      * Redraw the map with a new snapshot.
      * @param {MapSnapshot} snapshot
      */
-    private redraw(snapshot: MapSnapshot) {
-        this.map.nodes.clear()
+    private redraw(snapshot: MapSnapshot, backup: boolean = false) {
```

Comment:
so when is this called? currently its not? its always false, isn't it?

Reply:
I forgot to add `this.redraw(snapshot, true)` to the snapshot import. :D 

This is only called during a snapshot import right now.